web_01.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="./css/style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+        <title>Website_02: Mini</title>
+    </head>
+
+    <body>
+        <header>
+            <div class="main__header" id="main_header">
+                <div class="container">
+                    <div class="header__logo">
+                        <a href="#"><img src="./images/logo.png" alt=""></a>
+                    </div>
+                    <div class="header__menu">
+                            <nav>
+                                <ul>
+                                    <li><a href="#">HOME +</a>
```

Comment:
`+` sign can be achieved by adding class with `content: '+';` property instead of hardcoding it.